City-of-Helsinki/helsinki-paatokset
# PP-314: Made misc accessibility fixes

Ticket: https://helsinkisolutionoffice.atlassian.net/browse/PP-314

**How to test**

Checkout branch and run `drush cr`

Go to some policymaker page i.ex. https://helsinki-paatokset.docker.so/fi/paattajat/kaupunginvaltuusto

- The members listing should now have a alt="" if there is an image
- The link has now a translatable aria label
-The name tag is now h3

Go to all initiatives page and check that the "Valitse vuosi" text is now inside a span

Go to https://helsinki-paatokset.docker.so/fi/paattajat

The accordion titles should now be h3

Go to front page and add news element and upcoming meetings paragraphs if they aren't there already.

Check that the large news liftup has an empty alt in the image and that there is a aria-label on the link that says "Siirry uutiseen: [Otsikko]"

The upcoming meetings paragraph items should now be h3 instead of h4
